front/src/service.ts

```diff
@@ -416,8 +437,55 @@ export class ScratchyService {
      * @return {Promise} Empty promise.
      */
     async deleteMessage(messageID: string): Promise<void> {
-        const response = await axios.delete(this.apiUrl + "/message/" + messageID);
+        const response = await axios.delete(this.url + this.apiNamespace + "/message/" + messageID);
         this.messageCache.delete(messageID);
         return response.data;
     }
-}
\ No newline at end of file
+
+/* Call server with Websocket */
+
+     /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @param {Room} id  - room id , eg:60895dd62d1a706830c31f10
+     * @returns {Promise<undefined>} -
+     */
+    async join(roomId:Room): Promise<void> { 
+        this.socket.emit('join',roomId)
+    }
+
+         /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @param {string} id  - room id , eg:60895dd62d1a706830c31f10
```

Comment:
not room id but user id

Reply:
Oups

Reply:
fixed here [9dc6391](https://github.com/Draeggon/scratchy/pull/2/commits/9dc6391613ec668a8d0dbf1f7c84f9311c4b1c72)